applications/launchpad_v2/src/styles/Icons/Password1.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import { SVGProps } from 'react'
+
+const SvgPassword1 = (props: SVGProps<SVGSVGElement>) => (
+  <svg
+    width='1em'
+    height='1em'
+    viewBox='0 0 22 22'
+    fill='none'
+    xmlns='http://www.w3.org/2000/svg'
+    {...props}
+  >
+    <path
+      d='M22 11A11 11 0 0 0 11 0v2.22A8.78 8.78 0 0 1 19.78 11H22Z'
+      fill='#A2281D'
+    />
+  </svg>
```

Comment:
You can add this icon to Icons tests.
You need to add here the `data-testid='svg-password1`.
And then, in `styles/Icons/Icons.test.tsx`, add `Password1` to:

```
const ICONS = [
  ...
  'Password1'
   ...
]
```